orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -218,18 +233,36 @@ def test_forward_call_multiple_positional_arguments():
     output = ort_model(x, y)
     assert output is not None
 
-# TODO: Re-enable after "Support models with dynamically defined inputs" done.
-# def test_forward_call_positional_arguments():
-#     device = 'cuda'
+def test_forward_call_multiple_positional_arguments_var_keyword():
+    device = 'cuda'
 
-#     N, D_in, H, D_out = 64, 784, 500, 10
-#     model = NeuralNetPositionalArguments(input_size=D_in, hidden_size=H, num_classes=D_out).to(device)
-#     model = ORTModule(model)
-#     args = [torch.randn(N, D_in, device=device), torch.randn(N, D_in, device=device), torch.randn(N, D_in, device=device)]
+    N, D_in, H, D_out = 64, 784, 500, 10
+    model = NeuralNetMultiplePositionalArgumentsVarKeyword(input_size=D_in, hidden_size=H, num_classes=D_out).to(device)
+    with self.assertWarns(SomeWarning) as warning_context_manager:
+        ort_model = ORTModule(model)
+    self.assertIn('ortmodule.py', warning_context_manager.filename)
+    self.assertEqual(221, warning_context_manager.lineno)
```

Comment:
Are we hard coding the line where the warning will come from in the testing? If that is the case, this will break CI almost at every PR